photoprism/Makefile

```diff
@@ -11,13 +11,14 @@ config-hook:
 	@${BIN}/reconfigure_ask ${ENV_FILE} PHOTOPRISM_TRAEFIK_HOST "Enter the photoprism domain name" photoprism${INSTANCE_URL_SUFFIX}.${ROOT_DOMAIN}
 	@${BIN}/reconfigure ${ENV_FILE} PHOTOPRISM_INSTANCE=$${instance:-default}
 	@${BIN}/reconfigure_htpasswd ${ENV_FILE} PHOTOPRISM_HTTP_AUTH default=no
-	
+	@${BIN}/reconfigure_password ${ENV_FILE} PHOTOPRISM_DATABASE_PASSWORD 32
+
 	@echo ""
 	@${BIN}/reconfigure_ask ${ENV_FILE} PHOTOPRISM_ADMIN_USER "Enter the admin user name" ${PHOTOPRISM_ADMIN_USER}
 	@echo ""
-	@${BIN}/reconfigure_ask ${ENV_FILE} PHOTOPRISM_ADMIN_PASSWORD "Enter the admin password" ${PHOTOPRISM_ADMIN_PASSWORD}
+#	@${BIN}/reconfigure_ask ${ENV_FILE} PHOTOPRISM_ADMIN_PASSWORD "Enter the admin password" ${PHOTOPRISM_ADMIN_PASSWORD} 
```

Comment:
I'm thinking that if the admin password is an initial password only, and that the user needs to change it after its loaded, then this can't really be changed later from the .env, so it should just be a static password (ie 'password'), documented, and tell the user to change it in the interface instead. Otherwise its confusing that the password is here in the .env but it can't really be changed here. (If I understand it correctly)

Reply:
Yeah, you got it right. I changed it how you suggested.